src/main/java/org/kohsuke/github/GHPullRequestReviewComment.java

```diff
@@ -78,12 +81,19 @@ public String getPath() {
         return path;
     }
 
-    public int getPosition() {
-        return position;
+    @CheckForNull
+    public Integer getPosition() {
+        return position == -1 ? null : position;
+    }
+
+    @CheckForNull
+    public Integer getOriginalPosition() {
+        return original_position == -1 ? null : original_position;
     }
 
-    public int getOriginalPosition() {
-        return originalPosition;
+    @CheckForNull
+    public Long getInReplyToId() {
+        return in_reply_to_id == -1 ? null : in_reply_to_id;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)